shell/common/common_hive.sh

```diff
@@ -233,7 +235,7 @@ prepare_hive_config() {
     $DSH "mkdir -p '$(get_hive_conf_dir)' '$(get_local_bench_path)/hive_logs'; cp -r '$(get_local_configs_path)/hive$(get_hive_major_version)_conf_template'/* '$(get_hive_conf_dir)'"
 
     $DSH "$(get_perl_exports)
-    /usr/bin/perl -i -pe '$(get_hive_substitutions)' '$HIVE_SETTINGS_FILE' '$(get_hive_conf_dir)'/*"
+    /usr/bin/perl -i -pe '$(get_hive_substitutions $1)' '$HIVE_SETTINGS_FILE' '$(get_hive_conf_dir)'/*"
```

Comment:
Missing quotes around `'$1'` - since this is a human-readable "display name", we should support writing it however we want.